podcasts/Sharing/ShareImageView.swift

```diff
@@ -24,46 +24,74 @@ enum ShareImageStyle: CaseIterable {
             return "small"
         }
     }
+
+    var videoSize: CGSize {
+        switch self {
+        case .large:
+            CGSize(width: 292, height: 438)
+        case .medium:
+            CGSize(width: 292, height: 293)
+        case .small:
+            CGSize(width: 324, height: 169)
+        }
+    }
 }
 
 struct ShareImageView: View {
 
     let info: ShareImageInfo
     let style: ShareImageStyle
 
+    @Binding var angle: Double
+
     var body: some View {
         ZStack {
             LinearGradient(gradient: info.gradient, startPoint: .top, endPoint: .bottom)
+            let rotationFactor = sin(Angle(degrees: angle).radians)
+            KidneyShape()
+                .fill(info.gradient.stops.first?.color ?? .black)
+                .blur(radius: 50)
+                .opacity(0.15 + 0.5 * abs(rotationFactor))
+                .offset(x: -50, y: 0)
+                .rotationEffect(.degrees(180 * rotationFactor))
+            KidneyShape()
+                .fill(.white)
+                .blur(radius: 50)
+                .offset(x: 50, y: 0)
+                .opacity(0.15 + 0.5 * abs(rotationFactor))
+                .rotationEffect(.degrees(-180 * rotationFactor))
+                .blendMode(.softLight)
             Color.black.opacity(0.2)
             switch style {
             case .large:
                 VStack(spacing: 32) {
                     image()
-                        .frame(width: 200, height: 200)
+                        .aspectRatio(1, contentMode: .fit)
```

Comment:
Since sizes are now defined, I preferred to define an aspect ratio here.

Reply:
Could the aspect ratio value be connected to the enum: .large, .medium, ... and be return from there?